src/Utilities/useProvisioningPermissions.js

```diff
@@ -0,0 +1,27 @@
+import { usePermissions } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
+
+const useProvisioningPermissions = () => {
+  const { hasAccess: hasAWSAccess, isLoading: awsLoading } = usePermissions(
+    'provisioning',
+    ['provisioning:*:*', 'provisioning:reservation.aws:write']
+  );
+  const { hasAccess: hasAzureAccess, isLoading: azureLoading } = usePermissions(
+    'provisioning',
+    ['provisioning:*:*', 'provisioning:reservation.azure:write']
+  );
+  const { hasAccess: hasGCPAccess, isLoading: gcpLoading } = usePermissions(
+    'provisioning',
+    ['provisioning:*:*', 'provisioning:reservation.gcp:write']
```

Comment:
There is also `provisioning:reservation:write` I'm guessing that most likely means, the user has all permissions, but I'd better be safe here :)

Reply:
Interesting, is it a  bug?
Looks like `provisioning:reservation:write` is not a general permission.
There are `Launch on AWS/Azure/GCP` roles that add the `provisioning:reservation:*` permission,  therefore if I added `provisioning:reservation:write` as a general permission in the array, and I have only `Launch on AWS` role, I can also launch GCP / Azure instances.